Tasks/FtpUploadV2/tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+    "compilerOptions": {
+        "target": "ES6",
+        "module": "commonjs",
+        "skipLibCheck": true
```

Comment:
consider enabling `"strict": true` or some flags from that family.  "Strict null checks" and "no implicit any" are the most useful.